src/app/models/animal/animal.model.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { TestBed } from '@angular/core/testing';
+
+import { AnimalCategory } from 'enums/animal-category.enum';
+
+import { AnimalModel } from './animal.model';
+
+describe('Animal Model', () => {
+  let model: AnimalModel;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [AnimalModel]
+    });
+
+    model = TestBed.get(AnimalModel);
+  });
+
+  it('should return true if animal is from water', () => {
+    const water = AnimalCategory.water;
```

Comment:
Can you use directly `AnimalCategory.water`, I dont think assigning into a variable is really necessary.